scripts/LVGLImage.py

```diff
@@ -819,6 +819,16 @@ def _png_to_alpha_only(self, cf: ColorFormat, filename: str):
                 rawdata += row
 
         self.set_data(cf, w, h, rawdata)
+    
+    def sRGB_to_linear(self, x):
+        if x < 0.04045:
+            return x/12.92
+        return pow((x+0.055)/1.055, 2.4)
+
+    def linear_to_sRGB(self, y):
+        if y <= 0.0031308:
+            return 12.92 * y
+        return 1.055 * pow(y, 1/2.4) - 0.055
 
     def _png_to_luma_only(self, cf: ColorFormat, filename: str):
```

Comment:
Is the luma here should be called lumi(nance) instead?

Reply:
Yeah, luma and luminance are different terms:
https://www.geeksforgeeks.org/difference-between-luma-and-luminance/
But honestly I don't know which is the correct name in our case.